build.sbt

```diff
@@ -2,26 +2,27 @@ name := "spark-connector"
 
 organization := "com.couchbase.client"
 
-version := "1.2.1"
+version := "1.2.2"
 
 description := "Official Couchbase Spark Connector"
 
 organizationHomepage := Some(url("http://couchbase.com"))
 
-scalaVersion := "2.10.5"
+scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.11.8", "2.10.6")
+crossScalaVersions := Seq("2.11.8", "2.11.0")
```

Comment:
that doesn't make much sense, we gotta keep 2.10 as well the default is correctly 2.11 above anyways